pfcli/service/client/checkpoint.py

```diff
@@ -226,6 +227,7 @@ def upload_files(
                             local_path,
                             url_dict,
                             t,
+                            max_workers // 2,
```

Comment:
Same with mpu cases.
I have tested with uploading 10 files each with 5GB, but changing `max_worker // 2` to `1` or `max_worker` does not show any changes in file uploading throughput.
Could you tell me if there is anything that I should keep in mind for testing?

Reply:
It seems that the file upload throughput is strongly bound to the internet environment. When I evaluated the throughput in my home, it showed over 50MB/s throughput when using more threads, but when I do the same evaluation in my workplace, the throughput is bound to 20MB/s.